### ISSUE TITLE
Breaks with latest conda-build >= 24.1.0

### ISSUE THREAD

**Issue by paul-sud** [chrisjbillington/setuptools-conda#18]
Hi there,

It looks like the latest version of `conda-build` ([24.1.0](https://github.com/conda/conda-build/releases/tag/24.1.0)) no longer works with this package. The last known working version is [3.28.3](https://github.com/conda/conda-build/releases/tag/3.28.3). The reason is described in https://github.com/conda/conda-build/issues/5167 - invoking `conda-build` results in a non-zero return code, which raises an error when called here: https://github.com/chrisjbillington/setuptools-conda/blob/cdcc241b466f7015a886bda8b6585ba162790556/setuptools_conda/setuptools_conda.py#L706 . Thus, the packages are never copied to `conda_packages`.

I fixed this by pinning `conda-build` on my end but a better fix would be to update to `conda build` as `conda-build` is being deprecated.

**Comment by chrisjbillington** [comment]
Thanks for letting me know. Looks like this specific problem may be fixed: https://github.com/conda/conda-build/pull/5169

But I'll take a look at `conda build` when I get the time, since I've been a bit unhappy with what happens when you install conda-buid into the non-base environment (https://github.com/conda/conda-build/issues/3813 - don't know for sure if this is still an issue but I don't know of it being fixed). Perhaps the switch to a subcommand of `conda` (normally only installed to the base environment) is addressing that.

**Comment by paul-sud** [comment]
Yes that does look like the fix for the `conda-build` exit status, thanks for linking! Feel free to close this issue if you plan on addressing the `conda build` upgrade elsewhere.

**Comment by chrisjbillington** [comment]
I don't recall if I did anything to address this or if it resolved itself via upstream changes, but setuptools-conda currently works with newer conda-build versions, so closing!
